specification/rpc/connect-rpc.md

```diff
@@ -0,0 +1,58 @@
+# Semantic Conventions for Connect RPC
+
+**Status**: [Experimental][DocumentStatus]
+
+The Semantic Conventions for [Connect](http://connect.build) extend and override the [RPC spans](rpc-spans.md) and [RPC metrics](rpc-metrics.md) Semantic Conventions
+that describe common RPC operations attributes in addition to the Semantic Conventions
+described on this page.
+
+## Connect RPC Attributes
+
+`rpc.system` MUST be set to `"connect_rpc"`.
+
+Below is a table of attributes that SHOULD be included on client and server Connect RPC measurements.
```

Comment:
Hum I'm not sure if it's just me but I feel it's a bit unclear if the attributes here are relevant for spans, metrics or logs, or to all of them. I know they are for metrics and spans because I compared the old pages but I wonder if readers will be confused by this. The word `measurements` is maybe the problem? 


Also: how will this will look like when we generate the metrics via YAML?. Will they all be under `rpc-metrics` and here we only keep this table? Or do we repeat all the metrics again here? If we don't repeat, I don't know if it's valuable to split them like this, given people will have to go "back and forth" to read the rcp metrics + attributes for the specific technology 🤔

Reply:
For such pages (that will contain all signals) maybe the layout needs to be a bit different? Like 

```
# Semantic Conventions for Connect RPC

## Spans

...

## Metrics

when metrics are coming from YAML, we will have each of them individually like
### `rpc.client.duration`

## Logs
...
```
I feel this would be clearer. Let's see what the others think.

Reply:
> If we don't repeat, I don't know if it's valuable to split them like this, given people will have to go "back and forth" to read the rcp metrics + attributes for the specific technology

I think we cannot avoid going "back and forth" (duplicating general attributes everywhere). The alternative is to keep everything in one file per signal (as it is now). 
But the problems with that are:

- Technology specific semantic conventions are spread across different files (trace, metrics, logs). 
- Even though general and technology specific attributes for a signal would be on the same page, it's still sort of a "back and forth" (with scrolling) as the page tends to get huge the more specific technologies are added.

I like your proposal of structuring the tech-specific files. 

We should also keep in mind that we are discussing this as a general structuring principle (so not just for RPC). For databases, for example, there are way more different specific technologies, and I feel there a tech-based structuring is very helpful in navigation.

Reply:
> Technology specific semantic conventions are spread across different files (trace, metrics, logs).

Not with your proposed structured right? Things like `aws-lambda.md`, or `mongodb` will be on their own file, with all the signals inside. So a "single source of truth" I guess. 

> Even though general and technology specific attributes for a signal would be on the same page, it's still sort of a "back and forth" (with scrolling) as the page tends to get huge the more specific technologies are added.

Fair, the page will indeed get long. But at least to navigate it's "easier" with the help of a toc and sections. The only "back and forth" that we will have is scrolling but no way to avoid that unless we break each tech per signal.. and _some_ back and forth for the "general" attributes that are defined in other pages. But I assume that also there's no workaround. 

What I'm concerned is: ALL metrics for RPC or Databases being in `rpc-metrics`, `database-metrics` and then the tech-individual files just reference to that. There's pros and cons on both approaches, I guess we just have to decide. 

I raised similar concerns in this PR https://github.com/open-telemetry/semantic-conventions/pull/93#discussion_r1242417643 where I'm porting the RPC metrics to YAML. Now we have a chance of:

A - Duplicate the attributes on ALL metrics
B - Leave the attributes in a table, and just mention they ALL apply to ALL RPC metrics. 

It would then be similar to the individual pages like `gRPC`, `connect`. They would just say: The metrics are in `rpc-metrics` and all of them apply here. That would require people to move back and forth. And then if later tech-specific metrics are defined it becomes even worse - they will have to reference two files.

Reply:
as discussed in the SemConv WG, will deal with that in a follow up